### PR TITLE
[RA2 Ch6]: Reformulate gaps and adding ingress and egress

### PR DESCRIPTION
**Pull request by CsatariGergely**
All gaps are reformulated to use the template.
6.2.4 reformulated to contain the ingress, egress and load balancers.

closes: #1340 
closes: #1341

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter06.md [hunk 1]
```diff
@@ -50,39 +50,54 @@ While this Reference Architecture is being developed, Gaps will be identified th
 <a name="6.2.3"></a>
 ### 6.2.3 Kubernetes as a VM-based VNF Orchestrator
 
-> In order to support a transition from VNFs only to VNFs and CNFs in the same environment.
+> **Related requirements:** None.
+
+> **Baseline project:** _Kubernetes v1.18.6_, _Kubevirt v0.30.5_
+
+> **Gap description:** Kubernetes and at least on CRI compliant runtime shuld support the runing of VNF-s without completly changing the VNF-s architecture and deployment artifacts.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter06.md:57]:
```suggestion
> **Gap description:** Kubernetes and at least one CRI compliant runtime should support the running of VNFs without requiring changes to the VNF's architecture and deployment artifacts.
```

File: doc/ref_arch/kubernetes/chapters/chapter06.md [hunk 2]
```diff
@@ -50,39 +50,54 @@ While this Reference Architecture is being developed, Gaps will be identified th
...
 
 
 <a name="6.2.4"></a>
 ### 6.2.4 Multiple network interfaces on Pods
 
-> As well as having multiple network interfaces on Pods (e.g. Multus), need to support different network interfaces in different Pods using different CNI plugins within the same cluster.
+> **Related requirements:** [RM Chapter 4.2.2](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications)
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter06.md:63]: [RM Chapter 4.2.2](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications) doesn't really require multiple network interfaces. What is stated is:
`Note, the number of virtual network interfaces, aka vNICs, associated with a virtual compute instance, is directly related to the number of vNIC extensions declared for the environment.`

I think we need a specific requirement for this in RA2 Ch02.

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter06.md:63]: We already have a list of required interface setups in [2.2.2 Virtual Network Interface Specifications](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter02.md#222-virtual-network-interface-specifications) where each requires multiple interface support.

**Comment by tomkivlin** [comment]
@pgoyal01 @cntt-n/ref-a2 could you review again please?

File: doc/ref_arch/kubernetes/chapters/chapter06.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_arch/kubernetes/chapters/chapter06.md:63]: @CsatariGergely These are not about multiple interfaces but rather the different speeds that can be selected for an interface.

Comment (petorre) [reply doc/ref_arch/kubernetes/chapters/chapter06.md:63]: Maybe improve the old text if needed, but not delete it.

**Comment by tomkivlin** [comment]
Idle for 4 days - merging.

**Pull request merged**